skore/tests/integration/test_cross_validate.py

```diff
@@ -157,3 +161,19 @@ def test_plot_cross_validation():
         ),
     }
     plot_cross_validation(cv_results)
+
+
+def test_aggregated_cross_validation(rf, in_memory_project):
+    args = rf
+    cross_validate(*args, project=in_memory_project)
+    assert isinstance(
+        in_memory_project.item_repository.get_item("cross_validation_aggregated"),
+        CrossValidationAggregationItem,
+    )
+
+    cross_validate(*args, project=in_memory_project)
+    cross_validate(*args, project=in_memory_project)
+    cross_validate(*args, project=in_memory_project)
+    in_memory_project.item_repository.get_item(
+        "cross_validation_aggregated"
+    ).plot.show()
```

Comment:
Its a bit strange, at the top you test the item type, at the bottom you test that the plot can be shown.
What are you trying here?